projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css

```diff
@@ -256,6 +256,12 @@
 	}
 }
 
+/* Prevent box-shadow at the top of the sidebar */
+#adminmenu li.menu-top:hover,
+#adminmenu a.menu-top:hover {
+	box-shadow: none;
+}
+
```

Comment:
I am afraid that we will need to be more specific cause this removes the hover effect from all top level menu items
![](https://cln.sh/BtfeXg+)

Reply:
Good catch - thank you! Fixed! :)